docs/tools/testing/testing_greentea.md

```diff
@@ -178,6 +314,190 @@ This detects your attached target and drives the test. If you need to rerun the
 
 For an explanation of the arguments used in this command, please run `mbedhtrun --help`.
 
+### Command-line usage
+
+This section highlights a few of the capabilities of the Greentea command-line interface. For a full list of the available options, please run `mbed test --help`.
+
+#### Listing all tests
+
+You can use the `-l` argument to list all available tests:
+
+```
+$ mbed test -l
+[mbed] Working path "/Users/janjon01/repos/first-greentea-test" (program)
+Test Case:
+    Name: mbed-os-components-storage-blockdevice-component_flashiap-tests-filesystem-fopen
+    Path: ./mbed-os/components/storage/blockdevice/COMPONENT_FLASHIAP/TESTS/filesystem/fopen
+Test Case:
+    Name: mbed-os-features-cellular-tests-api-cellular_device
+    Path: ./mbed-os/features/cellular/TESTS/api/cellular_device
+
+...
+```
+
+#### Executing all tests
+
+The default action of Greentea using `mbed test` is to execute all tests that are found. You can also add `-v` to make the output more verbose.
+
+#### Limiting tests
+
+You can select test cases by name using the `-n` argument. This command executes all tests named `tests-mbedmicro-rtos-mbed-mail` from all builds in the test specification:
+
+```
+$ mbed test -n tests-mbedmicro-rtos-mbed-mail
+```
+
+When using the `-n` argument, you can use the `*` character at the end of a test name to match all tests that share a prefix. This command executes all tests that start with `tests-mbedmicro-rtos`:
+
+```
+$ mbed test -n tests-mbedmicro-rtos-*
+```
+
+You can use a comma (`,`) to separate test names (argument `-n`) and build names (argument `-t`). This command executes the tests `tests-mbedmicro-rtos-mbed-mail` and `tests-mbed_drivers-c_strings` for the `K64F-ARM` and `K64F-GCC_ARM` builds in the test specification:
+
+```
+$ mbed test -n tests-mbedmicro-rtos-mbed-mail,tests-mbed_drivers-c_strings -t K64F-ARM,K64F-GCC_ARM
+```
+
+#### Selecting platforms
+
+You can limit which boards Greentea uses for testing by using the `--use-tids` argument.
+
+```
+$ mbed test --use-tids 02400203C3423E603EBEC3D8,024002031E031E6AE3FFE3D2
+```
+
+Where `02400203C3423E603EBEC3D8` and `024002031E031E6AE3FFE3D` are the target IDs of platforms attached to your system.
+
+You can view target IDs using the [`mbed-ls`](https://github.com/ARMmbed/mbed-os-tools/tree/master/packages/mbed-ls) tool (installed with Greentea).
```

Comment:
Please use `mbed detect` instead.

Reply:
Updated. Copied it from old content but missed this.